README.rst

```diff
@@ -0,0 +1,199 @@
+|Build Status| |FOSSA Status Shield Badge|
```

Comment:
Does RST files view on github?

Reply:
@CosmicWebServices , yes, for example, see [**my fork**](https://github.com/Kristinita/Pyfancy-2/tree/SashaGoddess).

Thanks.